ivy/data_classes/container/experimental/linear_algebra.py

```diff
@@ -1889,3 +1889,138 @@ def tucker(
             prune_unapplied=prune_unapplied,
             map_sequences=map_sequences,
         )
+
+    @staticmethod
+    def static_tt_matrix_to_tensor(
+        tt_matrix: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+        /,
+        *,
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
```

Comment:
the type hints seem to be wrong here- would you please look at some other functions in this file and update this?